broadway/api/utils/bootstrap.py

```diff
@@ -168,6 +168,10 @@ def initialize_app(
                 ),
                 client_handlers.CourseWorkerNodeHandler,
             ),
+            (
+                r"/api/v1/queue_length/{}".format(id_regex.format("course_id")),
```

Comment:
I think it would make sense for the endpoint to be
`/api/v1/queue/{course_id}/length`. later we could add more features like `/api/v1/queue/{course_id}/position` etc.